### ISSUE TITLE
Why is the nowadays activity so quiet here?

### ISSUE THREAD

**Issue by danifornells** [haiticss/haiticss#84]
Maybe some of your are having same question, so let me break the silence.
HaitiCSS started as a side project building a multi-brand design system, and the early idea was to be an extended version of INUIT or ITCSS that should fit our case. However, on the way, we discovered our best input on conceptual abstractions, and not on code itself. For us is even hard to see HaitiCSS as a tool/boilerplate or as a methodology. While we have different points of view and goals, the project is maturing on our usage case and not here, our apologies for that. The main reason that pointed us to do it, was the fragility and frequency of the changes on the project, too noise to follow. Concluding, the project is still alive, but unfortunately not here for now. I promise to come back with revamped code and ideas soon ;)

**Comment by syther101** [comment]
Not related but I just wanted to thank you for your work thus far. I've been tasked with untangling our CSS and after jumping all around the web reading multiple articles on ITCSS, discovering multiple open source repos on GitHub (include those by Harry himself) I came across haiticss just yesterday. 

Of all the examples I have come across thus far it is the one I have found myself agreeing with the most.

I have to say it almost feels like the spiritual successor to inuitcss. While I understand ITCSS is just a way of thinking it helps a whole bunch to have a base to build from and solidify understanding.

I particularly like your inclusion of the foundations layer, particularly the way you have separated the spaces from being based on a single core-spacing-unit like in Harry's examples. The article linked in issue #64 was particularly interesting.

Anyway long story short. Keep up the good work all!
